examples/internal/clients/generateunboundmethods/model_protobuf_any.go

```diff
@@ -10,6 +10,5 @@
 package generateunboundmethods
 
 type ProtobufAny struct {
-	TypeUrl string `json:"typeUrl,omitempty"`
-	Value string `json:"value,omitempty"`
+	Type_ string `json:"@type,omitempty"`
```

Comment:
`swagger-codegen` now generates go code which I wouldn't expect to work. Really I'd expect this type to not be generated, and any usages of `ProtobufAny` actually be of type `interface{}`, and say it's up to the caller to provide a value which marshals to a JSON object which is valid under the rules set out by the protojson package. 

I guess this would be a swagger-codegen problem, and one which most likely exists across most programming languages. Not sure how to approach this.